lib/core/services/auth-guard-sso-role.service.ts

```diff
@@ -19,24 +19,34 @@ import { Injectable } from '@angular/core';
 import { JwtHelperService } from './jwt-helper.service';
 import { ActivatedRouteSnapshot, CanActivate, Router } from '@angular/router';
 import { MatDialog } from '@angular/material/dialog';
+import { PeopleContentService } from './people-content.service';
+import { PersonEntry } from '@alfresco/js-api';
 
 @Injectable({
     providedIn: 'root'
 })
 export class AuthGuardSsoRoleService implements CanActivate {
+    acsAdminRole = 'ACS_ADMIN';
```

Comment:
this code doesn't belong to the guard it should be part of the service.
The guard should always call the service, if the service that should store the info to avoid calls